src/main/java/mServer/crawler/sender/MediathekArd.java

```diff
@@ -236,82 +252,101 @@ private void filmSuchen1(String strUrlFeed, String thema, boolean weiter) {
             } else {
                 return;
             }
-            while (!Config.getStop() && (pos = seite1.indexOf(MUSTER_FILM_SUCHEN1, pos)) != -1) {
+             while (!Config.getStop() && (pos = seite1.indexOf(MUSTER_FILM_SUCHEN1, pos)) != -1)
+            {
                 ++count;
-                if (!CrawlerTool.loadLongMax()) {
-                    if (count > 5 && !thema.equalsIgnoreCase("alpha-Centauri")) {
+                if (!CrawlerTool.loadLongMax())
+                {
+                    if (count > 5 && !thema.equalsIgnoreCase(THEMA_ALPHA_CENTAURI))
+                    {
                         break;
                     }
                 }
                 pos += MUSTER_FILM_SUCHEN1.length();
                 url = seite1.extract("documentId=", "&", pos);
-                if (url.contains("\"")) {
-                    url = url.substring(0, url.indexOf('\"'));
+                if (url.contains("\""))
+                {
+                    url = url.substring(0, url.indexOf("\""));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Put single-quotes around '\"' to use the faster "indexOf(char)" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3027)